---
title: 3303 - ReceiveContextAbandonWithException
ms.date: 03/30/2017
ms.assetid: eae255f8-13fd-4d8e-9f9e-56aea0d588c3
ms.openlocfilehash: 377cfdb9012ac3386c8bbc376efcfc7f65bb769c
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 05/04/2018
ms.locfileid: "33459572"
---
# <a name="3303---receivecontextabandonwithexception"></a><span data-ttu-id="a8355-102">3303 - ReceiveContextAbandonWithException</span><span class="sxs-lookup"><span data-stu-id="a8355-102">3303 - ReceiveContextAbandonWithException</span></span>
## <a name="properties"></a><span data-ttu-id="a8355-103">属性</span><span class="sxs-lookup"><span data-stu-id="a8355-103">Properties</span></span>  
  
|||  
|-|-|  
|<span data-ttu-id="a8355-104">ID</span><span class="sxs-lookup"><span data-stu-id="a8355-104">ID</span></span>|<span data-ttu-id="a8355-105">3303</span><span class="sxs-lookup"><span data-stu-id="a8355-105">3303</span></span>|  
|<span data-ttu-id="a8355-106">关键字</span><span class="sxs-lookup"><span data-stu-id="a8355-106">Keywords</span></span>|<span data-ttu-id="a8355-107">通道</span><span class="sxs-lookup"><span data-stu-id="a8355-107">Channel</span></span>|  
|<span data-ttu-id="a8355-108">级别</span><span class="sxs-lookup"><span data-stu-id="a8355-108">Level</span></span>|<span data-ttu-id="a8355-109">信息</span><span class="sxs-lookup"><span data-stu-id="a8355-109">Information</span></span>|  
|<span data-ttu-id="a8355-110">通道</span><span class="sxs-lookup"><span data-stu-id="a8355-110">Channel</span></span>|<span data-ttu-id="a8355-111">Microsoft-Windows-应用程序服务器-应用程序/调试</span><span class="sxs-lookup"><span data-stu-id="a8355-111">Microsoft-Windows-Application Server-Applications/Debug</span></span>|  
  
## <a name="description"></a><span data-ttu-id="a8355-112">描述</span><span class="sxs-lookup"><span data-stu-id="a8355-112">Description</span></span>  
 <span data-ttu-id="a8355-113">当接收上下文放弃失败并且具有异常时，将发出此事件。</span><span class="sxs-lookup"><span data-stu-id="a8355-113">This event is emitted when the receive context abandon failed with an exception.</span></span>  
  
## <a name="message"></a><span data-ttu-id="a8355-114">消息</span><span class="sxs-lookup"><span data-stu-id="a8355-114">Message</span></span>  
 <span data-ttu-id="a8355-115">%1 被放弃并且具有异常 %2。</span><span class="sxs-lookup"><span data-stu-id="a8355-115">%1 was Abandoned with exception %2.</span></span>  
  
## <a name="details"></a><span data-ttu-id="a8355-116">详细信息</span><span class="sxs-lookup"><span data-stu-id="a8355-116">Details</span></span>
